api_proxy.yaml

```diff
@@ -393,6 +416,22 @@ definitions:
       - name
       - preferred_email
       - version
+  PublicSession:
+    type: object
+    title: User session data
+    description: Describe the current session of an authenticated user.
+    properties:
+      expired:
```

Comment:
why we have both `expired` and `expireAt`, don't they tell the same thing?

Reply:
@fpersico do you remember why the app needs both?

Reply:
I don't think it is a requirement, expireAt is enough.